automatic/selenium.powershell/legal/VERIFICATION.txt

```diff
@@ -0,0 +1,17 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+The module has been downloaded from <https://www.powershellgallery.com/packages/Selenium>
+and can be verified by following these steps:
+
+1. Save the module from the PowerShell Gallery:
+  - Save-Module -Name Selenium -Version 3.0.1
+2. Get the checksum for each file using one of the following methods:
+  - Using powershell function 'Get-FileHash'
+  - Use chocolatey utility 'checksum.exe'
+3. The checksums should match the following:
+
+  file name: LICENSE
+  checksum type: SHA256
+  checksum: BAEB4A5B8A2F782E21C9F508EEBC728ADBA739F7B7FB8BA3FF07747A4C9FE296
```

Comment:
This indicates that the checksum is for the License file, is that correct?
We typically do not include that checksum for the license.

The location of the embedded license is usually included differently as well, typically at the bottom of the file with something like the following
```
File 'LICENSE.txt' is obtained from <https://github.com/git-for-windows/git/blob/703601d6780c32d33dadf19b2b367f2f79e1e34c/COPYING>
```

Reply:
I think in this case it's because that version of the license file is included in the output saved artifact you get from step 1, which is unusual-ish. It is one of the files that could be checked (like the README.md).

![image](https://github.com/chocolatey-community/chocolatey-packages/assets/1975761/a94e65fb-ec26-4c33-a471-0361b77bd5f2)

It seems valid to include that - otherwise we're removing stuff from the artifact you get from the publisher, or not mentioning it in the verification, which might raise questions if noticed.

Reply:
Technically I guess we're shipping that file twice in the package. I don't think that's problematic?

Reply:
In which case, that license file should probably be compared against the license we embed in the legal folder instead then.

Reply:
> I thought it was to the license file we had there, but if it isn't then the verification file is missing the location where that license was downloaded from and needs to be added.

I believe this is step 1 in the verification file: it's downloaded with the `Save-Module` command. Would it be sufficient to remove the `LICENSE.txt` from the `legal` directory and just use the one from the `Save-Module` command?

Reply:
I think we still need that license file for CCR moderation purposes, unless we mess with the zip file before pushing the package, but that's just a surface level thought - I'm still uneasy messing with the artifact.